src/utils/queryParamsStringify.test.ts

```diff
@@ -0,0 +1,20 @@
+import { queryParamsStringify } from './queryParamsStringify';
+
+describe('queryParamsStringify', () => {
+  test('should stringify array params correctly', () => {
+    const params = {
+      ids: ['1', '2', '3'],
+    };
+    expect(queryParamsStringify(params)).toEqual('ids=1&ids=2&ids=3');
+  });
+
+  test('should stringify array params correctly when there are other params', () => {
+    const params = {
+      ids: ['1', '2', '3'],
+      other: 'param',
+    };
+    expect(queryParamsStringify(params)).toEqual(
+      'ids=1&ids=2&ids=3&other=param'
+    );
+  });
```

Comment:
Can you give an example here that wouldn't have passed the generated code?

Reply:
None of these pass with the generated code. I don't fully understand why, because I agree the online resources I found made it seem like an issue limited to iframes.